extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -58,22 +58,11 @@ export class AmpScrollableCarousel extends BaseCarousel {
 
     this.container_ = this.element.ownerDocument.createElement('div');
     this.container_.classList.add('-amp-scrollable-carousel-container');
-    st.setStyles(this.container_, {
```

Comment:
scrollable carousel is not slides - 

There are 2 carousels type ="slides" type ="carousel"

You are editing 

Type="carousel" here and modifying type="slide" in the css.